modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/FieldBase/ReactFieldBase.es.js

```diff
@@ -315,9 +316,22 @@ function FieldBase({
 								})}
 								tabIndex="0"
 							>
-								{label && showLabel && label}
+								{label && showLabel && (
+									<>
+										{hideField ? (
+											<span className="text-secondary">
+												{label}
+											</span>
```

Comment:
Check hideField to print classname or not instead of render fragment

Reply:
Done. See it here https://github.com/liferay-forms/liferay-portal/pull/484/commits/4a319cce54f917cfecbd62c1bda54262e93cc6b2